deploy/scripts/install.sh

```diff
@@ -25,17 +26,29 @@ update_image_tags() {
     fi
     # Check if the image tag is latest - this is the default tag so no need to write
     if [[ "$image_tag" == "latest" ]]; then
-        echo "Using :latest image tag"
+        echo "Using :latest image tag for all containers"
         return
     fi
 
-    echo "Setting container image tags to: $image_tag"
+    # If image_tag is for a dev build based on the main branch, a matching image tag will not
+    # exist for the ui container.  In this case fall back to using the latest tag for the ui container.
+    # Tags for dev builds on the main branch look like 0.6.0-main-119-gf75bcff
+    local services=()
+    if [[ "$image_tag" =~ -main- ]]; then
+        services+=("api" "periodic" "worker")
+    else
+        services+=("api" "periodic" "worker" "ui")
+    fi
```

Comment:
Consider breaking this out to a function and add comments to the function. I think it will make maintenance clearer

 ```
 get_services_for_tag() {
    local image_tag="$1"